src/Homework_02/02_Examples(www.hackerrank.com).py

```diff
@@ -0,0 +1,102 @@
+"""
+1. Read an integer N.
+Without using any string methods, try to print the following:
+123...N
+Note that ... represents the values in between.
+"""
+
+
+if __name__ == '__main__':
+    n = int(input())
+    for i in range(1, n + 1):
+        print(i, end='')
+
+
+"""
+2. Read two integers from STDIN and print three lines where:
+The first line contains the sum of the two numbers.
+The second line contains the difference of the two numbers (first - second).
+The third line contains the product of the two numbers.
+1 <= a <= 10 ** 10
+1 <= b <= 10 ** 10
+"""
+
+
+if __name__ == '__main__':
+    a = int(input())
+    b = int(input())
+    if (1 <= a <= 10 ** 10) and (1 <= b <= 10 ** 10):
```

Comment:
redundant checks - it's put to string description

Reply:
![02](https://user-images.githubusercontent.com/58551098/71125595-53d07d80-21f8-11ea-9e9d-ee9164f624bf.jpg)
Task condition